policy/modules/services/dbus.te

```diff
@@ -312,13 +321,17 @@ corecmd_read_bin_files(session_bus_type)
 corecmd_read_bin_pipes(session_bus_type)
 corecmd_read_bin_sockets(session_bus_type)
 
-corenet_all_recvfrom_netlabel(session_bus_type)
-corenet_tcp_sendrecv_generic_if(session_bus_type)
-corenet_tcp_sendrecv_generic_node(session_bus_type)
-corenet_tcp_bind_generic_node(session_bus_type)
+tunable_policy(`dbus_can_network',`
+	allow session_bus_type self:tcp_socket { accept listen };
+
+	corenet_all_recvfrom_netlabel(session_bus_type)
+	corenet_tcp_sendrecv_generic_if(session_bus_type)
+	corenet_tcp_sendrecv_generic_node(session_bus_type)
+	corenet_tcp_bind_generic_node(session_bus_type)
 
-corenet_sendrecv_all_server_packets(session_bus_type)
-corenet_tcp_bind_reserved_port(session_bus_type)
+	corenet_sendrecv_all_server_packets(session_bus_type)
+	corenet_tcp_bind_reserved_port(session_bus_type)
+')
```

Comment:
This should move down above the optional block.

Reply:
Amended